packages/ui/components/dao/DaoCard.tsx

```diff
@@ -1,5 +1,129 @@
-export interface DaoCardProps {}
+import clsx from 'clsx'
+import Link from 'next/link'
+import { useTranslation } from 'react-i18next'
 
-export const DaoCard = ({}: DaoCardProps) => {
-  return <p>placeholder</p>
+import {
+  DaoCardMemberCheck,
+  DaoCardPin,
+  DaoCardProposals,
+  Governance,
+  SubDaoArrow,
+} from '@dao-dao/icons'
+
+export interface SubDaoInfo {
+  parentDaoHref: string
+  parentDaoImageUrl: string
+}
+
+export interface DaoCardProps {
+  name: string
+  description: string
+  imageUrl: string
+  established: Date
+  href: string
+
+  junoBalance: number
+  proposalCount: number
+
+  subDaoInfo?: SubDaoInfo
+
+  pinned: boolean
+  onPin: () => null
```

Comment:
```suggestion
  onPin: () => void
```

We don't want to require callbacks to return `null`